wiki/Internet_Relay_Chat/en.md

```diff
@@ -1,28 +1,25 @@
----
-needs_cleanup: true
----
-<!-- TODO: awkward terminology, some parts link to dated info, client-specific help should be updated with modern clients -->
-
 # Internet Relay Chat
 
-The [Internet Relay Chat](https://en.wikipedia.org/wiki/Internet_Relay_Chat), also known as IRC, is a well established standardized protocol for chatting with numerous clients available to connect with.
+*For further information, see [Internet Relay Chat (Wikipedia)](https://en.wikipedia.org/wiki/Internet_Relay_Chat "Wikiepdia")*
+
+The **Intenet Relay Chat** (**IRC**) is an application layer protocol for chatting with numerous clients available to connect with. Although not developed by/for osu!, it is an integral part of its infrastructure and to community [bots](/wiki/Bot_account).
 
 ## osu!Bancho
 
-osu!Bancho offers a gateway for IRC (in-game chat) access. You can connect with your own client and chat with other people without using the game client. Take note that this IRC protocol is modified therefore do not expect your client's features to work properly.
+*Notice to osu!Bancho users: this IRC protocol is modified, and thus it is not guaranteed that all of a clients' features will work properly on it. (E.g., [HexChat](https://hexchat.github.io/))*
```

Comment:
rather than "protocol is modified", how about saying it is a naive/partial implementation?